tests/test_parser.py

```diff
@@ -292,3 +292,25 @@ def test_implementation_constructor():
         entries2 = rispy.load(f, implementation=rispy.RisImplementation.WOK)
 
     assert entries1 == entries2
+
+
+def test_encodings():
+    fn = DATA_DIR / "example_utf_chars.ris"
+    p = Path(fn)
+
+    encoding1 = "ascii"
+    encoding2 = "utf-8"
+
+    with open(fn, "r", encoding=encoding2) as file:
```

Comment:
i think you can remove this 304-305

Reply:
The reason it's there is to compare the different methods of opening. There's a bug with read_text where it reads certain characters incorrectly, so this tests that. see: https://stackoverflow.com/questions/49428723

Reply:
although I guess I'm not sure this is something that's worth testing here

Reply:
@shapiromatron still want me to remove this part?

Reply:
i'd get rid of the variables encoding1 and encoding2 and just write in the load() method calls. then its gtg!